dubbo-metadata/dubbo-metadata-api/pom.xml

```diff
@@ -84,7 +88,61 @@
             <version>${project.parent.version}</version>
             <scope>compile</scope>
         </dependency>
-
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>${protobuf-java.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-rpc-triple</artifactId>
+            <version>${project.parent.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Is it necessary to import triple here?

Reply:
Why do I introduce this dependency here? The main reason is that the proto class file generated by dubbo-complier depends on three classes under the triple module. One solution is to move these three classes to dubbo-rpc-api, so that other modules may only need to introduce dubbo-rpc-api dependency if they need to introduce dubbo-compiler dependency in the future.

Reply:
Supplement: The purpose of introducing dubbo-compiler here mainly refers to the java classes that need to be used to generate proto file definitions.